packages/devtools_app/test/performance_model_test.dart

```diff
@@ -90,7 +91,10 @@ void main() {
               'vsyncOverhead': 1000
             },
           ],
-          PerformanceData.cpuProfileKey: goldenCpuProfileDataJson,
+          PerformanceData.cpuProfileKey: CpuProfileData.generateFromCpuSamples(
+            isolateId: goldenSamplesIsolate,
+            cpuSamples: CpuSamples.parse(goldenCpuSamplesJson)!,
+          ).toJson,
```

Comment:
isn't this equivalent to `goldenCpuProfileDataJson`? why not just continue using `goldenCpuProfileDataJson` directly?

Reply:
good catch. I forgot to switch this back, when I was fixing things up :)